nvflare/apis/job_def_manager_spec.py

```diff
@@ -162,7 +173,7 @@ def get_jobs_by_status(self, run_status: RunStatus, fl_ctx: FLContext) -> List[J
         """Gets Jobs of a specified status.
 
         Args:
-            run_status (RunStatus): status to filter for
+            run_status (RunStatus): status to filter for: a single or a list of status values
```

Comment:
it will be more clear if we also update the type hint here to be:
`Union[RunStatus, List[RunStatus]]`

Reply:
I'm scared of these type hints after running into Python's "TypeError: Subscripted generics cannot be used with class and instance checks".  This error happens in a totally unrelated place if you use List[Str] as type hint of an arg of an abstract class. It took me hours to nail this down.